src/api/streams/updateStream.js

```diff
@@ -2,7 +2,11 @@
 import type { ApiResponse, Auth } from '../apiTypes';
 import { apiPatch } from '../apiFetch';
 
-export default (auth: Auth, id: number, property: string, value: string): Promise<ApiResponse> =>
-  apiPatch(auth, `streams/${id}`, res => res, {
-    [property]: value,
-  });
+export type ParamsType = {
+  new_name?: string,
+  description?: string,
+  is_private?: boolean,
```

Comment:
Is this an exhaustive list?

Reply:
yup! stream have limited properties.

Reply:
`ParamsType` does not sound like concrete or a good name.
First, we don't name types `*Type` :)
Also, `Param`? It is a parameter/argument to a function but you can do better.

Reply:
any name you want to suggest? :)
`ApiBody`, `ApiParameters`??